tools/@aws-cdk/cfn-guard-custom-rules-tool/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'cfn-guard-custom-rules-tool'
+description: 'CFN Guard for custom or granular guard rules'
+author: QuantumNeuralCoder
+
+inputs:
+  data_directory:
+    description: "Path to CloudFormation templates"
+    required: true
+  rule_set_path:
+    description: "Path to a single .guard file locally"
+    required: false
+  rule_set_url:
+    description: "URL to a single .guard file on GitHub"
+    required: false
+  show_summary:
+    description: "cfn-guard summary output. Options are all, pass, fail, skip or none"
+    required: false
+    default: "fail"
+  output_format:
+    description: "cfn-guard output format. Options: json, yaml, single-line-summary"
+    required: false
+    default: "single-line-summary"
+
+runs:
```

Comment:
since cfn-guard is required, I believe it worth adding a step to install cfn-guard, so we do not need every time we use this GH action to add a step to install it